bindings/wasm/bindings.cpp

```diff
@@ -72,6 +73,13 @@ Manifold Transform(Manifold& manifold, std::vector<float>& mat) {
   return manifold.Transform(matrix);
 }
 
+Manifold Warp(Manifold& manifold, void* funcPtr) {
+  void (*f)(glm::vec3&) = reinterpret_cast<void (*)(glm::vec3&)>(funcPtr);
+  Manifold out = manifold.Warp(f);
+  EM_ASM({removeFunction($0)}, f);
```

Comment:
I was following this example: https://github.com/emscripten-core/emscripten/tree/main/test/interop

Reply:
I kind of feel like that the `removeFunction` call should be called in the binding code? So in case we want to reuse some warp with js functions, we can do so with minimal overhead. This is not really C++ code anyway.

Reply:
Good call, I was following that example a little too blindly.